src/Schema/Events/ImpressionTracking.ts

```diff
@@ -231,3 +232,30 @@ export interface SendOffersErrorMessage {
   collectors: number
   message: string
 }
+
+/**
+ * User sees the edit profile modal after sending an inquiry
```

Comment:
The user could also be shown the "add artist to my collection" modal (and close it without adding an artists) after sending an inquiry, so we'll need another impression event for that.